README.md

```diff
@@ -166,7 +168,68 @@ The airdrop has two rounds: In the first round, all participants will go through
+
+Round 4 follows a procedure similar to that of Round 3, with the exception of a different ranking method. Participants will receive **three** rounds of airdrops simultaneously around `2023-03-21 06:00 UTC`, and then proceed with the Interchain NFT Transfer according to the flow in the `Task Data`. For the final interchain transfer, they must first transfer the NFT back to their Iris addresses and then transfer it to the `last_owner`. In this round, the ranking is calculated by summing the differences in block heights of each NFT between the last NFT transfer and the first Interchain NFT transfer on IRISnet. If two participants have the same sum, the participant with the lower sum of the first transfer height of each NFT will rank higher. Therefore, if participants have team members, they can transfer the three NFTs in parallel to get higher rank. **Note that you must complete all of B5, B6 and B7 in order to be eligible for ranking.**
+
+**Task Data**
+```
+{
+  "type": "team race round",
+  "flow": "the flow id, check flow with flow-id on https://github.com/game-of-nfts/gon-testnets/blob/main/doc/flow-table.md",
+  "last_owner": "the ultimate owner of this NFT",
+  "start_height": "transfer before this height are considered invalid"
+}
+```
+
+**Rules:**
+
+- The `start_height` is the block height on IRISnet. Any transfer prior to the `start_height` will be considered invalid, which means your first Interchain NFT Transfer must happen after that height. Note that `start_height` is not used for calculate your grade.
+- The `last_owner` is an official address on IRISnet, and is the final recipient of the NFT in this game. 
+- Considering the time zone difference, the completion time is calculated by the **difference** in height between the last NFT transfer (to the `last_owner`) and the first Interchain NFT transfer (not `start_height`) on IRISnet.
+- Participants must use their own account addresses provided at the registration stage.
+- Participants must complete all of B5, B6 and B7 in order to be eligible for the ranking, which is calculated by summing the differences in block heights of each NFT between the last NFT transfer and the first Interchain NFT transfer on IRISnet.
+- The third round of the competition will conclude at `2023-03-24 23:59:59 UTC`. After this, you can still complete B5, B6 and B7, but your race rank will not be counted, and you will not be eligible to win B8.
+- The top 10 participants in each airdrop round will be awarded B8 points. Evidence is not required for B8.
+
+#### **Round 5: Quiz Game**
+
+In the quiz game, the organizers will airdrop 100 Quiz NFTs that are owned by 100 preset addresses around `2023-03-21 08:00 UTC`. Participants are required to **guess the answer** to the question in `Task Data`, which serves as the secret key to decrypt the mnemonics. The participants must then use the mnemonics to generate these preset addresses and transfer the NFT according to the flow.
```

Comment:
How do we find these NFTs? Will the addresses or class-id be revealed at the starting point?

Reply:
Yes, of course. Just the same as Round3. A list of these NFTs and their owner addresses will be published in a issue from gon-evidence.